tests/ChatTests/Direct.hs

```diff
@@ -1544,7 +1544,9 @@ testUserPrivacy =
       showActiveUser alice "alisa"
       -- connect using second user
       connectUsers alice bob
+      threadDelay 1000000
       alice #> "@bob hello"
+      threadDelay 1000000
```

Comment:
is second delay necessary?

Reply:
yes, because the time of received messages is rounded down and the order is not deterministic when sorted by time.